.github/CODEOWNERS

```diff
@@ -300,3 +300,6 @@ tests/js/spec/utils/profiling                                   @getsentry/profi
 /src/sentry/reprocessing2.py       @getsentry/owners-ingest @getsentry/owners-native
 
 ## End of Native ##
+
+# Profiling
+/src/sentry/api/endpoints/project_stacktrace.py @getsentry/profiling
```

Comment:
We already have a profiling section at https://github.com/getsentry/sentry/blob/master/.github/CODEOWNERS#L171-L177

Reply:
mind adding the profiling team to the `organization_stacktraces.py` endpoint as well? thanks!